datadog/dogstatsd/base.py

```diff
@@ -743,19 +854,32 @@ def event(
             string = "%s|#%s" % (string, ",".join(tags))
 
         if len(string) > 8 * 1024:
-            raise Exception(u'Event "%s" payload is too big (more than 8KB), ' "event discarded" % title)
+            raise Exception(
+                u'Event "{}" payload is too big (>=8KB). Event discarded'.format(
```

Comment:
```suggestion
                u'Event "{0}" payload is too big (>=8KB). Event discarded'.format(
```

Reply:
@jirikuncar Hmm - isn't `{0}` implied by `{}`? I don't mind changing it - it just seems a bit superfluous.